helpers/stylesheet.js

```diff
@@ -18,12 +19,19 @@ const factory = globals => {
 
         const url = cdnify(path);
 
-        let attrs = { rel: 'stylesheet' };
-
+        if (_.has(options.hash, 'resourceHint')) {
```

Comment:
lodash 🥲
you can try to do 
```suggestion
        if (options.hash.resourceHint) {
```